src/EditorFeatures/CSharpTest/Completion/CompletionProviders/InternalsVisibleToCompletionProviderTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Completion.Providers;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Roslyn.Test.Utilities;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Completion.CompletionProviders
+{
+    public class InternalsVisibleToCompletionProviderTests : AbstractCSharpCompletionProviderTests
+    {
+        private static readonly string s_keyPairFile = SigningTestHelpers.KeyPairFile;
+        private static readonly DesktopStrongNameProvider s_defaultProvider = new SigningTestHelpers.VirtualizedStrongNameProvider(ImmutableArray.Create<string>());
+        public InternalsVisibleToCompletionProviderTests(CSharpTestWorkspaceFixture workspaceFixture) : base(workspaceFixture)
+        {
+            // I needed to configure the workspace here, because CreateWorkspace was never called.
```

Comment:
*CompletionProviders/InternalsVisibleToCompletionProviderTests.cs* does not exisits anymore. These test were transfered to *EditorServicesTest2*.